app/pages/BeelieversAuction/BeelieversBid.tsx

```diff
@@ -124,6 +166,9 @@ export function BeelieversBid({ user }: BeelieversBidProps) {
 	});
 
 	const hasUserBidBefore = (user && user.amount !== 0) || false;
+	const bidInputInSUI = bidForm.watch("bid");
+	// TODO: get the highest bid
+	const highestBidInMist = 10 * 1e9;
```

Comment:
let's change the logic here - we want to get into the winning list, so we need to get above the Entry Price.
- rename variable
- update message